router/router.go

```diff
@@ -887,3 +894,33 @@ func (ctx *Router) embedTemplates() error {
 	}
 	return nil
 }
+
+// chooseTemplateByCustomTriggerType chooses template by message and output. Works only for servicenow templates.
+func chooseTemplateByCustomTriggerType(msg map[string]interface{}, outputType string) string {
+	template := ""
+
+	// choose template suffix by output
+	// Works only for servicenow templates
+	outputType = strings.ToLower(outputType)
+	if outputType != "servicenow" && outputType != "jira" {
+		return template
+	}
+
+	// choose template type by customTriggerType
+	trigger, ok := msg[customTriggerTypeField]
```

Comment:
Here you should check whether `resourceTypeKey` is equal to `"code-repository"` and apply it only for that per output

Reply:
Done